index.html

```diff
@@ -72,6 +72,27 @@
           <a>standardized payment method identifier</a>.
         </li>
       </ul>
+      <section>
+        <h3>
+          Validity
+        </h3>
+        <p>
+          The steps to <dfn>validate a payment method identifier</dfn> with a
+          <var>pmi</var> string are given by the following algorithm. It
```

Comment:
Makes sense, will fix here and then update globally. The other ones are badly written also, so will clean them up to match.